frontend/src/component/providers/AccessProvider/permissions.ts

```diff
@@ -1,6 +1,7 @@
 export const ADMIN = 'ADMIN';
 export const CREATE_FEATURE = 'CREATE_FEATURE';
 export const UPDATE_FEATURE = 'UPDATE_FEATURE';
+export const UPDATE_DEPENDENCY = 'UPDATE_DEPENDENCY';
```

Comment:
can we call it `UPDATE_FEATURE_DEPENDENCY`?

Reply:
sure, updating across all files to keep it consistent.